### PR TITLE
Add base_url to routes

### PR DESCRIPTION
**Pull request by davidbrochart**
Fixes https://github.com/jupyterlab/jupyterlab/issues/13463.

### CODE REVIEW

**Review by jtpio — APPROVED** [review]
Thanks!

**Review by hbcarlos — APPROVED** [review]
Thanks, @davidbrochart!

File: jupyter_server_ydoc/app.py [hunk 1]
```diff
@@ -62,7 +63,9 @@ def initialize_settings(self):
     def initialize_handlers(self):
         self.handlers.extend(
             [
-                (r"/api/yjs/roomid/(.*)", YDocRoomIdHandler),
-                (r"/api/yjs/(.*)", YDocWebSocketHandler),
+                url_path_join(
+                    self.settings["base_url"], (r"/api/yjs/roomid/(.*)", YDocRoomIdHandler)
```
Comment (fcollonval) [inline jupyter_server_ydoc/app.py:67]: I don't think this is correct as for example this is not what is done by Jupyter Server Terminals:

https://github.com/jupyter-server/jupyter_server_terminals/blob/574e1018b7d924f09c1ca45389182f1fd314caee/jupyter_server_terminals/app.py#L72

Comment (davidbrochart) [reply jupyter_server_ydoc/app.py:67]: Yes, and it seems to be confirmed by the Binder.

Comment (davidbrochart) [reply jupyter_server_ydoc/app.py:67]: I just yanked jupyter-server-ydoc v0.5.0 on PyPI.

**Pull request merged**